exporter/instanaexporter/config_test.go

```diff
@@ -28,27 +28,27 @@ func TestConfigValidate(t *testing.T) {
 	})
 
 	t.Run("Valid configuration", func(t *testing.T) {
-		c := &Config{Endpoint: "http://example.com/", AgentKey: "key1"}
+		c := &Config{Endpoint: "https://example.com/", AgentKey: "key1"}
 		err := c.Validate()
 		assert.NoError(t, err)
-
-		assert.Equal(t, "http://example.com/", c.Endpoint, "no Instana endpoint set")
 	})
 
 	t.Run("Invalid Endpoint Invalid URL", func(t *testing.T) {
-		c := &Config{Endpoint: "http://example.}~", AgentKey: "key1"}
+		c := &Config{Endpoint: "http://example.~}", AgentKey: "key1"}
```

Comment:
Is this invalid because of the http vs. https, or because of the chars at the end? The test is ambiguous and might fail for a different reason than intended.

Reply:
Good catch. This is probably some test I was running and forgot to roll it back.
This is not related to the https tests.